packages/react/index.classic.fb.js

```diff
@@ -57,4 +56,12 @@ export {
   useActionState,
   version,
 } from './src/ReactClient';
+
 export {jsx, jsxs, jsxDEV} from './src/jsx/ReactJSX';
+
+// export for backwards compatibility during upgrade
+export {useMemoCache as unstable_useMemoCache} from './src/ReactHooks';
+
+// export to match the name of the OSS function typically exported from
+// react/compiler-runtime
+export {useMemoCache as c} from './src/ReactHooks';
```

Comment:
I think @acdlite and @gnoff may have suggested the idea of moving the impl as well into it's own package and dynamically injecting it into the dispatcher iirc. This feels okay to me as a first step though

Reply:
I'm not sure that's possible as useMemoCache adds an extra field to the fiber.

I think what we can look into is how this should look in the server build. We might iterate on that as the server build should probably have a noop use memo cache or not even use the transform.